test/FunctionalTests/UnaryMethodTests.cs

```diff
@@ -337,6 +337,7 @@ static Task<Empty> ReturnContextInfoInTrailers(Empty request, ServerCallContext
         }
 
         [Test]
+        [Ignore("Regression in TestServer - https://github.com/aspnet/AspNetCore/issues/10737")]
```

Comment:
Seems like the issue has been resolved so disposing the request body no longer closes the stream.

Reply:
Fixing it by creating a new stream. This is more correct in line with how HttpClient+StreamContent is used.